packages/actor-rdf-parse-html-microdata/lib/ActorRdfParseHtmlMicrodata.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+import type { IActionRdfParseHtml, IActorRdfParseHtmlOutput } from '@comunica/bus-rdf-parse-html';
+import { ActorRdfParseHtml } from '@comunica/bus-rdf-parse-html';
+import type { IActorArgs, IActorTest } from '@comunica/core';
+import { MicrodataRdfParser } from 'microdata-rdf-streaming-parser';
+
+/**
+ * A comunica Microdata RDF Parse Html Actor.
+ */
+export class ActorRdfParseHtmlMicrodata extends ActorRdfParseHtml {
+  public constructor(args: IActorArgs<IActionRdfParseHtml, IActorTest, IActorRdfParseHtmlOutput>) {
+    super(args);
+  }
+
+  public async test(action: IActionRdfParseHtml): Promise<IActorTest> {
+    return true;
+  }
+
+  public async run(action: IActionRdfParseHtml): Promise<IActorRdfParseHtmlOutput> {
+    const mediaType = action.headers ? action.headers.get('content-type') : null;
+    // eslint-disable-next-line no-implicit-coercion
+    const xmlMode = !!mediaType && mediaType.includes('xml');
```

Comment:
If you don't want to disable the rule you can do `Boolean(mediaType && mediaType.includes('xml'))`, or even `Boolean(mediaType?.includes('xml'))`

Reply:
Thanks, good suggestions!